scripts/pkglib.sh

```diff
@@ -164,6 +164,33 @@ function checkout_repo()
     cp ../scripts/apport_hook.py $OBS_WORKDIR/$APPORT_HOOK
 }
 
+function generate_kernel_config() {
+    local kernel_version=$1
+    local kernel_filename=$2
+    local kernel_url=https://www.kernel.org/pub/linux/kernel/v4.x/linux-"${kernel_version}".tar.xz
```

Comment:
Again, Is there a way to avoid hard-coding `4.x` here I wonder? If not, can you create a variable for this (and also use it in `KR_SHA` which also hard-codes `4.x`).